src/main/frontend/db/model.cljs

```diff
@@ -1350,11 +1352,12 @@
 (defn get-all-properties
   []
   (let [properties (d/q
-                     '[:find [?p ...]
-                       :where
-                       [_ :block/properties ?p]]
-                     (conn/get-db))
-        properties (remove (fn [m] (empty? m)) properties)]
+                    '[:find [?p ...]
+                      :where
+                      [_ :block/properties ?p]]
+                    (conn/get-db))
+        properties (remove (fn [m] (or (empty? m) 
+                                       (gp-block/whiteboard-properties? m))) properties)]
```

Comment:
Yeah, the current implementation of putting all tldraw shape properties to `block/properties` is not being thought through indeed! 

IMO we have two types of `:block/properties`, one is managed by Logseq and is not meant to be used by the users, like these Tldraw shape properties and it may have no point that the user wants to run a query on them; other properties are user-facing ones like alias and tags.

In any case, if we want to put dynamic properties in `:block/properties`, we should filter them somehow with various levels of maintenance efforts in different places. Personally, I prefer no.2 which could make the data structure more explicitly expose its purpose.

Reply:
I also prefer no.3.

Reply:
@logseq-cldwalker @tiensonqin I believe we should make the decision asap. It will be getting harder to make the migration when this feature gets more users.

Reply:
Let's go with solution 1 for now.
![CleanShot 2022-09-29 at 11 25 23](https://user-images.githubusercontent.com/479169/192931699-e18433c2-92e8-4843-b0fe-2e2ceb11002f.png)

Reply:
Should I put all shape props into a single `:logseq.tldraw.shape` keyword, or renaming every prop key's with a prefixed namespace?

eg., 
```edn
{:logseq.tldraw.shape {:id "63346ddf-6c58-48c3-ae83-f6d1cbabd937"
                       :fill "black"}}
```
vs
```edn
{:logseq.tldraw.id  "63346ddf-6c58-48c3-ae83-f6d1cbabd937"
 :logseq.tldraw.fill "black"}
```

Reply:
I prefer the first solution here because tldraw shape's properties could be a lot compared to the outliner blocks, grouping all the shape props into a single `:logseq.tldraw.shape` keyword also makes it easy to handle when this block needs to be converted to another type in the future. 
The bad part is that it's impossible to query nested properties at this moment, for example `(property [:logseq.tldraw.shape :fill] "black")`, but we can address this later.

Reply:
Let's do this!

Reply:
An example edn after refactor:

```edn
{:blocks
 (#:block{:properties
          {:ls-type :whiteboard-shape,
           :logseq.tldraw.shape
           {:stroke "#ababab",
            :rotation 0,
            :borderRadius 2,
            :index 0,
            :scale [1 1],
            :ls-type "whiteboard-shape",
            :fill "var(--ls-secondary-background-color)",
            :type "box",
            :size [286.6484375 131.13671875],
            :strokeType "line",
            :strokeWidth 2,
            :opacity 1,
            :id "20c065e0-3faf-11ed-8659-3bc89882da7a",
            :noFill false,
            :point [452.74609375 212.99609375],
            :parentId "63346ddf-6c58-48c3-ae83-f6d1cbabd937",
            :nonce 1664425692188}}}
  #:block{:properties
          {:ls-type :whiteboard-shape,
           :logseq.tldraw.shape
           {:stroke "#ababab",
            :rotation 0,
            :borderRadius 2,
            :index 1,
            :scale [1 1],
            :ls-type "whiteboard-shape",
            :fill "var(--ls-secondary-b
ackground-color)",
            :type "box",
            :size [286.6484375 131.13671875],
            :strokeType "line",
            :strokeWidth 2,
            :opacity 1,
            :id "c9c958c0-3fc0-11ed-8215-9d39149053a1",
            :noFill false,
            :point [624.57421875 246.927734375],
            :parentId "63346ddf-6c58-48c3-ae83-f6d1cbabd937",
            :nonce 1664425692232}}}),
 :pages
 (#:block{:uuid #uuid "63346ddf-6c58-48c3-ae83-f6d1cbabd937",
          :properties
          {:ls-type :whiteboard-page,
           :logseq.tldraw.page
           {:id "63346ddf-6c58-48c3-ae83-f6d1cbabd937",
            :name "page",
            :bindings {},
            :nonce 141,
            :assets []}},
          :updated-at 1664433432479,
          :created-at 1664380383263,
          :type "whiteboard",
          :name "asdfasdfwwww",
          :original-name "asdfasdfwwww"})}
```

Reply:
Also prefer solution 1 as it made it easy to hide it from autocompletion

Reply:
No longer seeing query issue for new whiteboards 😄